packages/extension-chakra-storefront/src/theme/index.js

```diff
@@ -4,101 +4,138 @@
+import skeleton from './components/base/skeleton'
+import tooltip from './components/base/tooltip'
+import popover from './components/base/popover'
+//
+// // Project Component style overrides
+import appRecipe from './components/project/_app'
+// import Breadcrumb from './components/project/breadcrumb'
+import headerRecipe from './components/project/header'
+// import ListMenu from './components/project/list-menu'
+// import Footer from './components/project/footer'
+// import CheckoutFooter from './components/project/checkout-footer'
+// import LinksList from './components/project/links-list'
+// import DrawerMenu from './components/project/drawer-menu'
+// import NestedAccordion from './components/project/nested-accordion'
+// import LocaleSelector from './components/project/locale-selector'
+import offlineBannerRecipe from './components/project/offline-banner'
+// import Pagination from './components/project/pagination'
+// import ProductTile from './components/project/product-tile'
+// import SocialIcons from './components/project/social-icons'
+// import SwatchGroup from './components/project/swatch-group'
+// import ImageGallery from './components/project/image-gallery'
 
 // Please refer to the Chakra-Ui theme customization docs found
 // here https://chakra-ui.com/docs/theming/customize-theme to learn
 // more about extending and overriding themes for your project.
-export const overrides = {
-    styles,
-    layerStyles,
-    colors,
-    sizes,
-    space,
-    gradients,
-    shadows,
-    components: {
-        // base components
-        Accordion,
-        Alert,
-        Badge,
-        Button,
-        Checkbox,
-        Container,
-        Drawer,
-        FormLabel,
-        Icon,
-        Input,
-        Modal,
-        Popover,
-        Radio,
-        Select,
-        Skeleton,
-        Tooltip,
 
-        // project components
-        App,
-        Breadcrumb,
-        Header,
-        Footer,
-        CheckoutFooter,
-        LinksList,
-        ListMenu,
-        DrawerMenu,
-        NestedAccordion,
-        LocaleSelector,
-        OfflineBanner,
-        SocialIcons,
-        Pagination,
-        ProductTile,
-        SwatchGroup,
-        ImageGallery
-    }
-}
+export const overrides = defineConfig({
+    ...styles,
+    theme: {
+        layerStyles,
+        tokens: {
+            colors,
+            sizes,
+            gradients,
+            shadows,
+            fonts: {
+                heading: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`,
+                body: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`,
+                mono: `SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace`
+            }
+        },
+        semanticTokens: {
+            shadows
```

Comment:
Shadows are now semanticTokens. We are adding more shadows on top of what Chakra V3 has